api/contrib/envoy/extensions/compression/compressor/igzip/v3alpha/igzip.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package envoy.extensions.compression.compressor.igzip.v3alpha;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.compression.compressor.igzip.v3alpha";
+option java_outer_classname = "IgzipProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/compression/compressor/igzip/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Igzip Compressor]
+// [#extension: envoy.compression.compressor.igzip]
+
+message Igzip {
+  enum CompressionLevel {
+    option allow_alias = true;
+
+    DEFAULT_COMPRESSION = 0;
```

Comment:
same with [here](https://github.com/envoyproxy/envoy/pull/22651/files#diff-2ffc4413f13ea7bef53eaf69d44dee2fc34377ee740945d8d1e531158ecec9ebR33). And why we doesn't use standard here

Reply:
This will be cleaned up as above. Thanks!